spec/order_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/order'
+
+describe Order do
+  let(:menu) { double(:menu, dish: {:beef_burger => 4.0, :fish_burger => 2.0}) }
+  let(:menu_klass){ double(:menu_klass, :new => :menu) }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)